commonslibrary/settings/base.py

```diff
@@ -50,10 +50,13 @@
 ]
 
 THIRD_PARTY_APPS = [
+    'ckeditor',
     'raven.contrib.django.raven_compat',
```

Comment:
As you've installed it, could you add sorl here?

Probably won't impact you just yet.